cpp/examples/sde_seirvv.cpp

```diff
@@ -0,0 +1,98 @@
+/* 
+* Copyright (C) 2020-2024 MEmilio
+*
+* Authors: Nils Wassmuth, Rene Schmieding, Martin J. Kuehn
+*
+* Contact: Martin J. Kuehn <Martin.Kuehn@DLR.de>
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+#include <fstream>
+#include <vector>
+#include <iostream>
+
+#include "memilio/utils/logging.h"
+#include "sde_seirvv/model.h"
+#include "sde_seirvv/simulation.h"
+#include "memilio/utils/random_number_generator.h"
+
+int main()
+{
+    mio::set_log_level(mio::LogLevel::debug);
+
+    ScalarType t0   = 0.;
+    ScalarType tmid = 100.;
+    ScalarType tmax = 400.;
+    ScalarType dt   = 0.1;
+
+    mio::log_info("Simulating SEIRVV; t={} ... {} with dt = {}.", t0, tmax, dt);
+
+    mio::sseirvv::Model model;
+    
+    /*
+    * It is assumed that both variants have the same transmission probability 
+    * on contact and the same time exposed. The time infected is scaled by
+    * 1.35 for the second variant.
+    */
```

Comment:
```suggestion
    // It is assumed that both variants have the same transmission probability 
    // on contact and the same time exposed. The time infected is scaled by
    // 1.35 for the second variant.
```
Can you move this to the relevant section, i.e. line 70?